contrib/babelfishpg_tsql/runtime/functions.c

```diff
@@ -1187,3 +1192,85 @@ babelfish_integrity_checker(PG_FUNCTION_ARGS)
 
 	PG_RETURN_BOOL(true);
 }
+
+Datum
+bigint_degrees(PG_FUNCTION_ARGS)
+{
+	int64	arg1 = PG_GETARG_INT64(0);
+	float8	result;
+	 
+	result = DatumGetFloat8(DirectFunctionCall1(degrees, Float8GetDatum((float8) arg1)));
+
+	if (result < 0)
+		result = ceil(result);
+	else
+		result = floor(result);
+
+	 /* Range check */
```

Comment:
Since both int64 and float8 are 8 bytes, please add boundary test cases for max positive and negative values of bigint.